theorems/T000526.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: T000526
+if:
+  and:
+  - P000003: true
+  - P000042: true
+then:
+  P000043: true
+---
+
+Evident from {T240}, which shows that the {P37} base elements for the topology are {P38}.
```

Comment:
```suggestion
Evident from [Explore: $T_2$ ∧ Path connected ∧ ¬Arc connected](https://topology.pi-base.org/spaces?q=%24T_2%24+%2B+Path+connected+%2B+%7EArc+connected), which shows that the {P37} base elements for a {P3} topology are {P38}.
```

Future-proofing in case T240 changes in the future.

Reply:
Thanks for the edits!